chainer/functions/activation/tanh.py

```diff
@@ -57,13 +57,25 @@ def backward_gpu(self, x, gy):
 def tanh(x, use_cudnn=True):
     """Elementwise hyperbolic tangent function.
 
+     .. math:: f(x)=\\tanh(x).
+
     Args:
-        x (~chainer.Variable): Input variable.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
         use_cudnn (bool): If ``True`` and cuDNN is enabled, then this function
             uses cuDNN as the core implementation.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    .. admonition:: Example
+
```

Comment:
How about adding an example to show the concrete values of the outputs? For example,

```
It maps the input values into the range of :math:`[-1, 1]`.

>>> x = np.random.uniform(-3, 3, (3,)).astype('f')
>>> x
array([-2.43635702,  0.45567897,  2.57577729], dtype=float32)
>>> F.tanh(x).data
array([-0.98481113,  0.42655593,  0.98848587], dtype=float32)
```